### PR TITLE
update README instructions

### PR DESCRIPTION
**Pull request by evykassirer**

### CODE REVIEW

**Comment by julianguyen** [comment]
LGTM! Can you rebase from master? Also @lpatmo added a note on the PSQL errors, so you can remove that :P

**Comment by julianguyen** [comment]
Hey @evykassirer, any updates on this? :)

**Comment by evykassirer** [comment]
what did you want me to change? I updated some things at the meetup

File: README.md [hunk 1]
```diff
@@ -29,17 +29,15 @@ The app uses  **Ruby 2.3.1** and **Rails 4.2.6**. Please stick to these versions
 
 The steps below should be straightforward for Linux and macOS users. Windows users please refer to this [guide](https://gist.github.com/KelseyDH/11198922) for tips on setup.
 
-## Common Errors
-
-Commons errors faced during installation are documented in this [guide](https://github.com/julianguyen/ifme/blob/master/Errors.md).
+ERRORS: If you encounter an error during installation, please read your error messages and refer to this [guide](https://github.com/liirene1/ifme/blob/master/Errors.md) for help.
```
Comment (julianguyen) [inline README.md]: Please amend this line change!

File: README.md [hunk 2]
```diff
@@ -118,6 +118,11 @@ Install via [Homebrew](http://brew.sh/)
 
 `brew install postgresql`
 
+if you have a more recent version, you might need to downgrade:
```
Comment (julianguyen) [inline README.md]: Please remove these changes, we already have it documented in Errors.md!

**Comment by julianguyen** [comment]
Added comments on what needs to be changed! :P Thanks again!

**Comment by evykassirer** [comment]
thanks for clarifying! all updated :)

**Pull request merged**